app/templates/components/repository-sidebar.hbs

```diff
@@ -21,7 +21,7 @@
     <div>
       {{running-jobs jobs=runningJobs}}
     </div>
-    <hr class="sidebar-seperator">
+    <hr class="sidebar-separator">
```

Comment:
Depending on how you do things, this could be an API change. I'm assuming it isn't.